tests/test_file_utils.py

```diff
@@ -72,12 +73,26 @@ def gen_random_output():
         self.assertGreater(np.abs(out1 - out3).sum(), 0)
 
 
-def test_cached_path_extract(xz_file, tmp_path, text_file):
-    filename = xz_file
+def test_zstd_extractor(zstd_path, tmp_path, text_file):
```

Comment:
These tests should be skipped if the zstd lib is not installed

Reply:
The same as above...